x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render, fireEvent, act, screen } from '@testing-library/react';
+import React from 'react';
+import { CreateFieldButton } from './index';
+import {
+  indexPatternFieldEditorPluginMock,
+  Start,
+} from '../../../../../../../src/plugins/index_pattern_field_editor/public/mocks';
+
+import { TestProviders } from '../../../common/mock';
+import { useKibana } from '../../../common/lib/kibana';
+import { DataView } from '../../../../../../../src/plugins/data/common';
```

Comment:
If possible we want to mark these as `import type { DataView }` like we have in `main` right now: 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.test.tsx#L18

These two aren't 100% in sync right now. Can we do a small follow up to sync these up to where the files are either identical or near enough identical?